src/components/templates/OcTableFiles/OcTableFiles.vue

```diff
@@ -98,6 +98,7 @@
           :popper-options="popperOptions"
           mode="click"
           close-on-click
+          padding-size="remove"
```

Comment:
@kulmann do you remember why this has been done? Now the files list contextmenu has no padding and we may have to release ODS 11.3.1 for the web release 😬

Reply:
Yes, it's irritating for users that the context menu has a padding on its own, which makes some area around the menu items non-clickable. Ideally we should have the full menu item component area clickable in the future, this works towards it a little bit already.